gravitee-apim-console-webui/src/management/api/portal/ng-subscriptions/list/api-portal-subscription-list.component.spec.ts

```diff
@@ -294,7 +306,55 @@ describe('ApiPortalSubscriptionListComponent', () => {
       await creationDialogHarness.createSubscription();
       tick(400);
       const subscription = fakeSubscription();
-      expectApiSubscriptionsPostRequest(planV4.id, application.id, subscription);
+      expectApiSubscriptionsPostRequest(planV4.id, application.id, undefined, subscription);
+
+      expect(fakeUiRouter.go).toHaveBeenCalledWith('management.apis.ng.subscription.edit', { subscriptionId: expect.any(String) });
+
+      flush();
+    }));
+
+    it('should create subscription to a api key plan with customApiKey', fakeAsync(async () => {
+      const planV4 = fakePlanV4({ apiId: API_ID, generalConditions: undefined });
+      const application = fakeApplication({ id: 'my-app' });
+
+      await initComponent([], fakeApiV4({ id: API_ID, listeners: [] }), [planV4]);
+      const harness = await loader.getHarness(ApiPortalSubscriptionListHarness);
+
+      const createSubBtn = await harness.getCreateSubscriptionButton();
+      expect(await createSubBtn).toBeDefined();
+      expect(await createSubBtn.isDisabled()).toEqual(false);
+
+      await harness.openCreationDialog();
```

Comment:
Suggestion: change to createSubBtn.click ? Or move the checks that button is defined and enabled to the harness. 
I had to go to the harness to check if it openCreationDialog was clicking the same button that we check L323-326)